### ISSUE TITLE
"failed to locate parent configuration" on full reload

### ISSUE THREAD

**Issue by ricardovh** [ibissource/iaf#3126]
**❗ please do not add sensitive information in issues, you can provide extra information via email using issue number as reference ❗**

**Describe the issue**
When doing a full reload/restart of the application, I get issues with configurations that have a parentConfig, that it `failed to locate parent configuration`. Reloading the configuration manually after the full reload results in the parentConfig being found and the child being loaded.

It seems to depend on configuration.names, when I move all configurations behind the parent, it loads fine, but I cannot always influence this. For most applications this will not be an issue as they define the full list, but for applications with configurations.database.autoLoad it might be an issue, as running it locally with the Frank!Runner will result in an alphabetical list in configuration.names, which might have the parentConfig later in the order.

**Reporter**
Ricardo

**To Reproduce**

**Screenshots**

**Configuration**
```
Please provide the configuration of the Pipe or Receiver with the problem.
```

**Input**
```
Please provide an example of the input message. 
Alternatively, provide a Ladybug report.
```

**Environment**
```
FF! 7.7
FF! 7.8-20220426.180610
```

**Additional Environment**
 - DBMS: [e.g. Oracle, MSSQL, MariaDB, MySql, PostgreSQL]
 - Browser: [e.g. Chrome, Safari, Edge, Firefox]

**Additional Context**
Add any other context about the problem here. (f.e. ladybug report / test adapter with larva test)

**Comment by ricardovh** [comment]
Using configurations.database.autoLoad results in the same issue if the parent is alphabetically behind the child.

**Comment by ricardovh** [comment]
Further testing showed that explicitly setting `configurations.names` with just the parent config solves the issue as the parent is then forced to be loaded first before `configurations.database.autoLoad` loads the additional configurations from the database. Downside is that this breaks with the Frank!Runner as the Frank!Runner does not automatically determine the configurations to load when it detects configurations.names in the DeploymentSpecifics.

**Comment by nielsm5** [comment]
Has been fixed in #4033, closing this issue!
